testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -904,6 +905,7 @@
                                 <keycloak.singleUseObject.map.storage.provider>jpa</keycloak.singleUseObject.map.storage.provider>
                                 <keycloak.user.map.storage.provider>jpa</keycloak.user.map.storage.provider>
                                 <keycloak.userSession.map.storage.provider>jpa</keycloak.userSession.map.storage.provider>
+                                <auth.server.quarkus.mapStorage.profile.config>jpa</auth.server.quarkus.mapStorage.profile.config>
```

Comment:
How does it work when two active profiles specifies the same property to different values? 

It's possible to activate `map-storage-jpa` profile only when `map-storage` profile is activated. If there would be an issue with it, it'd be possible to duplicate properties from `map-storage` to `map-storage-jpa` to avoid activation of both of them.

Reply:
https://davidmarquis.wordpress.com/2011/01/13/maven-custom-properties-precedence/

> The order of precedence is based on the last profile in which this property is defined, in alphabetical order of profile name.

So in case of `map-storage,map-storage-jpa`, `auth.server.quarkus.mapStorage.profile.config` will be `jpa`